rust/threshold-schnorr/src/schnorr_example_rust/tests/tests.rs

```diff
@@ -33,14 +37,19 @@ fn test_impl(pic: &PocketIc, algorithm: SchnorrAlgorithm) {
     // Make sure the canister is properly initialized
     fast_forward(&pic, 5);
 
-    let message_hex = hex::encode("Test message");
+    // a message we can reverse to break the signature
+    // currently pocket IC only supports 32B messages for BIP340
```

Comment:
This limitation has been fixed now (not sure if it's a pocketic release though)

Reply:
the last release was on Sep 13, so probably not yet in pocketic